packages/yourloops/components/alarm/alarms-content-configuration.tsx

```diff
@@ -340,7 +241,11 @@ function AlarmsContentConfiguration(props: AlarmsContentConfigurationProps): JSX
           2. {t('severe-hypoglycemia')}
         </Typography>
         <Typography variant="caption" className={classes.categoryInfo}>
-          {t('current-trigger-setting-hypoglycemia', { hypoThreshold: hypoThreshold.value, veryLowBg: veryLowBg.value })}
+          {t('current-trigger-setting-hypoglycemia', {
```

Comment:
The fact that I don't see a data-testid here (which is ok as it is not needed), makes me wonder whether or not you have tested this label... :slightly_smiling_face: